tests/cli/test_asm.py

```diff
@@ -168,28 +174,28 @@ def test_add_seeds_from_stdin(self):
                 )
             ],
         )
-
+        # Actual call
         cli_main()
 
-    @patch(
-        "builtins.open",
-        new_callable=mock_open,
-        read_data=json.dumps(
-            [{"value": "1.1.1.1"}, {"value": "192.168. 0.15/24", "type": "CIDR"}]
-        ),
-    )
-    @patch(
-        "argparse._sys.argv",
-        [
-            "censys",
-            "asm",
-            "add-seeds",
-            "-i",
-            "test.json",
-        ]
-        + CensysTestCase.asm_cli_args,
-    )
-    def test_add_seeds_from_file(self, mock_file):
+    def test_add_seeds_from_file(self):
+        # Mock
+        self.patch_args(
+            [
+                "censys",
+                "asm",
+                "add-seeds",
+                "-i",
+                "test.json",
+            ],
+            asm_auth=True,
+        )
+        self.mocker.patch(
+            "builtins.open",
+            new_callable=mock_open,
```

Comment:
I think we should use self.mocker.mock_open instead

```suggestion
            new_callable= self.mocker.mock_open,
```